app/mapquest.properties

```diff
@@ -1 +1 @@
-api_key=Cmjtd|luua2qu2nd,b5=o5-gzb0
+api_key=YOUR_API_KEY
```

Comment:
let's use:
```
api_key=[PUT_YOUR_API_KEY_HERE]
```
since that's how it's described in the "getting started" docs, thx!